scripts/specializations/CpCourseManager.lua

```diff
@@ -190,6 +191,14 @@ function CpCourseManager:setFieldWorkCourse(course)
     course:setVehicle(self)
 end
 
+--- Copy the fieldwork course from another vehicle.
+function CpCourseManager:cpCopyCourse(vehicle)
```

Comment:
I'd call this `cpCopyCourseFrom(otherVehicle)` just to make it clear what it does